src/components/nodeColors.js

```diff
@@ -28,7 +28,7 @@ export const poolColor = '#f7f7f7';
 
 export function getDefaultNodeColors(node, color) {
   if (color) {
-    return { fill: Color(color).lighten(0.55).hex(), stroke: color };
+    return { fill: tinycolor(color).lighten(35).toHex8String(), stroke: color };
```

Comment:
After switching to `tinycolor`, using 55 for `lighten` made the background almost white for some colours, so I decided to stick with 35 instead.

Reply:
👍